src/render/draw_fill_extrusion.js

```diff
@@ -62,6 +62,7 @@ function drawExtrusionTiles(painter, source, layer, coords, depthMode, stencilMo
     const image = patternProperty.constantOr((1: any));
     const crossfade = layer.getCrossfadeParameters();
     const opacity = layer.paint.get('fill-extrusion-opacity');
+    const ao = [layer.paint.get('fill-extrusion-ambient-occlusion-intensity'), layer.paint.get('fill-extrusion-ambient-occlusion-radius')];
```

Comment:
If the question is about styling the property using object as property value instead of separated properties, it was reconsidered and discarded as it would depart from current approach for transitionable properties.